var/spack/repos/builtin/packages/pdc/package.py

```diff
@@ -14,29 +14,35 @@ class Pdc(CMakePackage):
     metadata operations to find data objects."""
 
     homepage = "https://pdc.readthedocs.io/en/latest/"
-    url      = "https://github.com/hpc-io/pdc/archive/refs/tags/0.1.tar.gz"
+    url      = "https://github.com/hpc-io/pdc/archive/refs/tags/0.2.tar.gz"
+    git      = "https://github.com/hpc-io/pdc.git"
 
     maintainers = ['houjun', 'sbyna']
 
+    version('0.2', sha256='2829e74da227913a1a8e3e4f64e8f422ab9c0a049f8d73ff7b6ca12463959f8b')
     version('0.1', sha256='01b4207ecf71594a7f339c315f2869b3fa8fbd34b085963dc4c1bdc5b66bb93e')
 
+    version('stable', branch='stable')
+    version('develop', branch='develop')
+
     conflicts('%clang')
-    depends_on('libfabric')
+    depends_on('libfabric sockets=fabricss,tcp,udp,rxm')
```

Comment:
This restriction seems unlikely: are you sure this is a limitation with the PDC software itself?

Reply:
For some systems, when running we got errors without specifying the fabrics option. The syntax was wrong there for some reason. Fixed it with the latest push.

Reply:
Can you describe the errors? Unless you're sure that it's a limitation with PDC itself, it's better to specify inability to build with a restrictive `conflicts` than a potentially over-broad solution -- and it's always good to document what the error is so that other people can try to reproduce it.

Reply:
It requires some fabrics, for instance, without the rmx in my clean Ubuntu I can build but not run the application. By default that fabric is not enabled, which causes the runtime error in the dependency library.

```
# [17393.553368] mercury->fatal: [error] /tmp/jlbez/spack-stage/spack-stage-mercury-2.1.0-mpq5l3oeuqqimzo2uoqy7vaorl47qbgo/spack-src/src/na/na_ofi.c:1800
 # na_ofi_provider_check(): Requested OFI provider "tcp;ofi_rxm" (derived from "tcp"
   protocol) is not available. Please re-compile libfabric with support for
   "tcp;ofi_rxm" or use one of the following available providers:
   udp tcp sockets
# [17393.553404] mercury->fatal: [error] /tmp/jlbez/spack-stage/spack-stage-mercury-2.1.0-mpq5l3oeuqqimzo2uoqy7vaorl47qbgo/spack-src/src/na/na.c:305
 # NA_Initialize_opt(): Specified class name does not support requested protocol
Error with HG_Init()

```

Reply:
After talking to our team, we decided to have a variant where we explicitly enable all fabrics and keep the default ones as not all systems will require those.